cmd/brimcap/ztests/analyze-process-readall.yaml

```
@@ -0,0 +1,36 @@
+script: |
+  mkdir wd1; mv noread.sh wd1
+  mkdir wd2; mv readall.sh wd2
+  bash gen.sh | brimcap analyze -config=config.yaml - | zq -z 'count()' -
+
+inputs:
+  - name: alerts.pcap
+  - name: config.yaml
+    data: |
+      analyzers:
+        - cmd: bash
+          args: [noread.sh]
+          name: noread
+          globs: ["*.json"] # so ztail will not try to read noread.sh
+          workdir: wd1
+        - cmd: bash
+          args: [readall.sh]
+          name: readall
+          globs: ["*.zson"]
+          workdir: wd2
+  - name: gen.sh
+    data: |
+      for i in {1..10000}; do
+        echo "{x: $i}"
+      done
+  - name: noread.sh
+    data: |
+      exit 0
+  - name: readall.sh
+    data: |
+      cat - > readall.zson
```

Feeding non-pcap data to `brimcap analyze` is surprising but fine I suppose.

But maybe shorten this?

```suggestion
script: |
  seq 10000 | brimcap analyze -config=config.yaml - | zq -z 'count()' -

inputs:
  - name: config.yaml
    data: |
      analyzers:
        - cmd: 'true'
          name: noread
        - cmd: bash
          args: [-c, 'cat > readall.zson']
          name: readall
```